topics/graalvm.md

```diff
@@ -0,0 +1,36 @@
+[//]: # (title: Gradle Application plugin)
+
+<microformat>
+<p>
+<control>Sample project</control>: <a href="https://github.com/ktorio/ktor-samples/tree/main/graalvm">graalvm</a>
+</p>
+</microformat>
+
+Ktor Server applications can make use of [GraalVM](https://graalvm.org) in order to have native images for different platforms, and of course,
+take advantage of the faster start-up times and other benefits that GraalVM provides. 
+
+Currently, Ktor Server applications that want to leverage GraalVM have to use CIO as the application engine. 
```

Comment:
A [sample](https://github.com/ktorio/ktor-samples/tree/main/graalvm) has the following info about engines:
`Using the Netty engine is not compatible with GraalVM. Please following the corresponding issue for updates.`

Here we have `...have to use CIO as the application engine`. Am I correct that we support all except Netty? If yes, I'd wrote it here.

Reply:
No. We currently don't support jetty either.